### ISSUE TITLE
An error produced in scrub_sample_GFF_junctions.py

### ISSUE THREAD

**Issue by linglingtingfei** [Magdoll/cDNA_Cupcake#23]
Hi @Magdoll ,
I used the script `scrub_sample_GFF_junctions.py ` of the latest version of Cupcake, but an error happened. Could you help me to solve it? I'd appreciate that.  I attached the screenshot of error at below.

![qq 20171208173449](https://user-images.githubusercontent.com/20789616/33763030-d562c50c-dc49-11e7-8c26-084069e550b9.jpg)

**Comment by Magdoll** [comment]
Hi @linglingtingfei ,

Looks like the `num_sample` field in your file "AD_Lab.junction_detail.txt" might not be a number. Can you double check that the `num_sample` column looks right? If it looks fine and are all integer numbers, you may have to send me the input files for testing. You can email it to etseng@pacb.com.

--Liz

**Comment by linglingtingfei** [comment]
Hi @Magdoll 
Thanks for the hint, I found out the reason. There is a little problem in the "AD_Lab.junction_detail.txt" file output by `summarize_sample_GFF_junctions.py`. It seems there is a defect in the program. The value for the column "label" is located in other raws. I also attached a screenshot of outputs.

![qq 20171212105520](https://user-images.githubusercontent.com/20789616/33866388-7009c5c6-df31-11e7-804b-0fbc108c3a4d.jpg)

**Comment by bioinformatist** [comment]
Dear @Magdoll ,
I also meet this problem, and below is my "junction_detail" file contents:
![image](https://user-images.githubusercontent.com/17194719/54926375-2990ae80-4f4b-11e9-87d5-5a8acd21e1cd.png)
It seems that the characters should be placed at "label" field came to newline in accident.
I'd like upload files needed in this step and help for reproducing this matter.
Can you help us with it?

The command I used is:
```shell
summarize_sample_GFF_junctions.py m54297_190108_070530.summarize.config m54297_190108_070530
```

Ahhhh... The additional file is too big for uploading. Is it convenient for providing me your email address?
Thank you in advance.

**Comment by Magdoll** [comment]
Can you update to the latest Cupcake version and first make sure the error is still reproduced?

**Comment by bioinformatist** [comment]
Yep, I'm sure that I do use the latest version of Cupcake for I just `git clone`d it and installed several days ago~

**Comment by bioinformatist** [comment]
@Magdoll The data used in the case is ~1 GB, I cannot upload them here.
Could you provide your email address if you're not mind?

**Comment by Magdoll** [comment]
@bioinformatist plz give me an email address and I'll send a file request

**Comment by bioinformatist** [comment]
@Magdoll My email address is suny226@mail2.sysu.edu.cn, thanks~

**Comment by Magdoll** [comment]
@bioinformatist request sent

**Comment by bioinformatist** [comment]
@Magdoll Hi, I just upload files. Plz check.
I have receive URL as:
[https://portal.pacificbiosciences.com/pickup.php?claimID=PgzF5cixRrM7DE4C&claimPasscode=zyK8TuSSZznX4v8r&emailAddr=etseng%40pacificbiosciences.com](https://portal.pacificbiosciences.com/pickup.php?claimID=PgzF5cixRrM7DE4C&claimPasscode=zyK8TuSSZznX4v8r&emailAddr=etseng%40pacificbiosciences.com)
And the command is `summarize_sample_GFF_junctions.py m54297_190108_070530.summarize.config m54297_190108_070530`

Thank you.

**Comment by bioinformatist** [comment]
@Magdoll Hi, any news about this problem?

**Comment by Magdoll** [comment]
Hi @bioinformatist ,

Apologies I was out for a while. This is now fixed and pushed to latest Cupcake version v7.0! Please give it a try.
-Liz
